packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -885,12 +892,59 @@ export class GCFBootstrapper {
     };
   }
 
+  private async getCloudRunUrl(
+    projectId: string,
+    location: string,
+    botName: string
+  ): Promise<string | null> {
+    // Cloud Run service names can only use dashes
+    const serviceName = botName.replace('_', '-');
+    const auth = new GoogleAuth({
+      scopes: ['https://www.googleapis.com/auth/cloud-platform'],
+    });
+    const authClient = await auth.getClient();
```

Comment:
You can actually just drop this line, and directly pass `auth` instead of fetching a client.

Reply:
I removed, then put it back because it's really hard to mock the auth here for environments that have ADC set up (like locally) and not (like GitHub actions CI).